packages/site/src/routes/create-dictionary.svelte

```diff
@@ -188,10 +188,17 @@
 
     <div class="mt-6" />
     <EditableGlossesField
+      admin
       availableLanguages={glossingLanguages}
       selectedLanguages={Array.from(glossLanguages)}
-      on:add={(e) => glossLanguages.add(e.detail.languageId)}
-      on:remove={(e) => glossLanguages.delete(e.detail.languageId)} />
+      on:add={(e) => {
+        glossLanguages.add(e.detail.languageId)
+        glossLanguages = glossLanguages
+      }}
+      on:remove={(e) => {
```

Comment:
@jacob-8 I fixed this because it didn't work due to how svelte handles reactivity inside event dispatcher: [here](https://svelte.dev/tutorial/updating-arrays-and-objects)